layrry-core/src/test/java/org/moditect/layrry/internal/resolver/FlatLocalRepositoryTest.java

```diff
@@ -15,6 +15,12 @@
  */
 package org.moditect.layrry.internal.resolver;
 
+import java.io.File;
```

Comment:
Is that ordering change required?

Reply:
Hmm shouldn't that be checked by impsort?

Reply:
Yeah, it should.